.github/workflows/run-example.yml

```diff
@@ -0,0 +1,66 @@
+on:
+    push:
+        paths:
+            - 'infrastructure/uxas/**'
+            - 'infrastructure/paths.sh'
+            - 'infrastructure/specs/**'
+            - 'infrastructure/run_example.py'
+            - 'anod'
+            - 'src/cpp/**'
+            - 'run-example'
+    pull_request:
+        paths:
+            - 'infrastructure/uxas/**'
+            - 'infrastructure/paths.sh'
+            - 'infrastructure/specs/**'
+            - 'infrastructure/run_example.py'
+            - 'anod'
+            - 'src/cpp/**'
+            - 'run-example'
+    workflow_dispatch:
+    schedule:
+        - cron: '20 3 * * 4'
```

Comment:
This will ensure that we run this periodically to recheck that run-example works.